crates/sui-graphql-rpc/src/types/transaction_block_effects.rs

```diff
@@ -56,21 +66,19 @@ pub(crate) type CUnchangedSharedObject = JsonCursor<usize>;
 pub(crate) type CObjectChange = JsonCursor<usize>;
 pub(crate) type CBalanceChange = JsonCursor<usize>;
 
+type CEvent = cursor::JsonCursor<usize>;
```

Comment:
nit: This guy's feeling lonely, being separated from his friends above.

Reply:
The visibility you've used here suggests to me that we can probably restrict the visibility of the other cursors? It would be helpful if you could try that out (not a blocker though).